br.gov.cti.invesalius.json

```diff
@@ -0,0 +1,203 @@
+    },
+    {
+        "name": "lapack",
+        "buildsystem": "cmake",
+        "builddir": true,
+        "config-opts": [
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DBUILD_SHARED_LIBS=ON",
+            "-DBUILD_TESTING=OFF",
+            "-DCMAKE_Fortran_COMPILER=/usr/lib/sdk/gfortran-62/bin/gfortran",
+            "-DLAPACKE=ON",
+            "-DCBLAS=ON"
+        ],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "http://www.netlib.org/lapack/lapack-3.8.0.tar.gz",
+                "sha512": "17786cb7306fccdc9b4a242de7f64fc261ebe6a10b6ec55f519deb4cb673cb137e8742aa5698fd2dc52f1cd56d3bd116af3f593a01dcf6770c4dcc86c50b2a7f"
+            }
+        ],
+        "cleanup": [ "/lib/cmake" ]
+    },
+    "shared-modules/glu/glu-9.0.0.json",
+    "python3-Pillow.json",
+    "python3-wxpython.json",
+    "python3-numpy.json",
+    "python3-scipy.json",
+    "python3-cython.json",
+    "python3-setuptools.json",
+    "python3-setuptools_scm.json",
+    "python3-python-dateutil.json",
+    "python3-matplotlib.json",
+    "python3-scikit-image.json",
+    "python3-bz2file.json",
+    "python3-nibabel.json",
+    "python3-pyserial.json",
+    "python3-psutil.json",
+    "python3-configparser.json",
+    {
+        "name": "runner",
+        "buildsystem": "simple",
+        "build-commands": [
+            "python3 setup.py build_ext --inplace",
+            "mkdir -p ${FLATPAK_DEST}/invesalius",
+            "cp -r * ${FLATPAK_DEST}/invesalius/",
+            "rm -rf /app/invesalius/build",
+            "rm -rf /app/invesalius/contrib",
+            "rm -rf /app/invesalius/po",
+            "rm -rf /app/invesalius/Dockerfile",
+            "rm -rf /app/invesalius/docs/devel",
+            "rm -rf /app/invesalius/docs/user_guide_en_source",
+            "rm -rf /app/invesalius/docs/user_guide_figures",
+            "rm -rf /app/invesalius/docs/user_guide_pt_BR_source",
+            "find /app/invesalius/ -name *.pyx -delete",
+            "find /app/invesalius/ -name *.pxd -delete",
+            "find /app/invesalius/ -name *.cpp -delete",
+            "find /app/invesalius/ -name *.c -delete",
+            "find /app/invesalius/ -name *.*ignore -delete",
+            "find /app/invesalius/ -name *.git -delete",
+            "install -D runner.sh /app/bin/runner.sh"
+        ],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://github.com/invesalius/invesalius3/archive/v3.1.99992.tar.gz",
+                "sha256": "d72e7f5957504ace5688edc0f083a11889c9d4a0937db75669379853501c32de"
+            },
+            {
+                "type": "file",
+                "path": "runner.sh"
+            }
+        ]
+    },
+    {
+        "name":"metadata",
+        "buildsystem":"simple",
+        "build-commands":[
+            "install -Dm644 invesalius.desktop /app/share/applications/br.gov.cti.invesalius.desktop",
```

Comment:
You could rename the files and use `install -t`.

Reply:
Done!